### DISCUSSION TITLE
"Ranged monsters no longer retreat when walking close to them"

### DISCUSSION THREAD

**Discussion by Zaratul555** [diasurgical/DevilutionX#5408] [Ideas]
Since Apocrypha fork is propably not gonna be updated more im requesting this (optional) feature. It would be a great QoL fix for meele chars.

**Comment by julealgon** [comment]
> ... QoL fix for meele chars.

Feels more like a massive AI simplification and trivialization of the game to me. 

Do you realize how (even more) easy the game would become with a change such as this?

**Comment by AJenbo** [comment]
I don't see why the Apocrypha wouldn't get anymore updates? In any case I think this would be to much of a modificatin compared to what the goals of DevilutionX is.

**Comment by Zaratul555** [comment]
Apocrypha author said that his mod is no longer gonna be updated. As for my suggestion: its not about making game easier\harder. Its just makes the game not irritating when fighting with ranged monsters.
Also game has lot of gameplay toggleables(shrines, potions) so why not implement this? For ppl wanting vanilla gameplay: just not turn this on.

**Reply by qndel** [reply]
because we aren't going to implement every single toggle people request, the game is already easy, that would be dumbing it down even more ... you can always make a fork and add whatever you want yourself 🤷

**Reply by julealgon** [reply]
> As for my suggestion: its not about making game easier\harder. Its just makes the game not irritating when fighting with ranged monsters.

It might not be _about_ making the game easier, but it _will_ make the game a LOT easier.

I understand your frustration with the vanilla mechanics, but this is not a good way to "solve" that problem without also requiring absurd amounts of balancing changes in the process.
